tests/unit/test_annotation.py

```diff
@@ -52,3 +52,18 @@ def fxt_hashkeys_diff(self):
     def test_compare_hashkey(self, fxt_hashkeys, expected, request):
         hashkey1, hashkey2 = request.getfixturevalue(fxt_hashkeys)
         assert (expected, hashkey1 == hashkey2)
+
+
+class RotatedBboxTest:
+    @pytest.fixture
+    def fxt_rot_bbox(self):
+        coords = np.random.randint(0, 180, size=(5,), dtype=np.uint8)
+        return RotatedBbox(coords[0], coords[1], coords[2], coords[3], coords[4])
+
+    @pytest.mark.parametrize("fxt_ann", ["fxt_rot_bbox"])
+    def test_create_polygon(self, fxt_ann, request):
```

Comment:
```suggestion
    def test_create_polygon(self, fxt_rot_bbox):
```
You can pass the name of the fixture as a parameter. `parametrize` decorator can also be removed.

Reply:
fixed at 2bbfbaeee81b4c8f89a67ccc937dce9c7c7d4aef